sedaro/README.md

```diff
@@ -162,6 +162,28 @@ with SedaroApiClient(api_key=API_KEY) as sedaro:
         assert str(e) == f'The referenced "BatteryCell" (id: {bc_id}) no longer exists.'
 ```
 
+### Multi-Block CRUD
+
+The `crud` method is also available for performing CRUD operations on multiple Sedaro blocks and/or root at the same time using kwargs as follows:
+- `root`: update fields on the root by passing a dictionary
+- `blocks`: create/update 1+ blocks by passing a list of dictionaries. If an `id` is present, the corresponding block will be updated. If an `id` isn't present, a `type` is required, and a new block will be created.
```

Comment:
Currently a type is always required.  I believe it will require nebula updates to disabled the validation in template_view.py.  Let's update this to always require a `type` for now and we can work to make it more flexible in the coming weeks.

Reply:
Oh yeah, nebula doesn't need `type`, but our temporary solution removing extra fields does need it.